_spinnaker/policy_engine.md

```diff
@@ -75,7 +80,10 @@ The Policy Engine supports the following OPA server deployments:
             args:
               - "run"
               - "-s"
-    ---
+```
+
+This ConfigMap exposes the OPA API: 
```

Comment:
Also this is not deploying a `ConfigMap`, is deploying a `Service`. The text could be changed to 
```
This Service exposes the OPA API: 
```